### ISSUE TITLE
Info.plist problem found

### ISSUE THREAD

**Issue by andyferriby** [BastiaanJansen/Brush#2]
Been wanting to test this app on my own apple watch as this is a great app, but some reason theres a problem with one of the info.plist files.

When the app loads on the xcode simulator, the first screen says Health Access "Brush WatchKit Extension" would like to access and update your Health data..... Surely this should be "Brush Watch Kit App"??? Then the second screen says the same "Brush WatchKit Extension" the watch is asking for permission to read the heath....This should also be the app name and not the extension? Thats why the app wont load on my watch its self?

**Comment by BastiaanJansen** [comment]
Does the app work on the simulator and not the real Apple Watch? Which WatchOS and iOS version are you running?

**Comment by andyferriby** [comment]
Yes the app works on simulator but the watch screen on simulator says" rush WatchKit Extension" would like to access and update your Health data.... This should be named as the watch app name, not the extension?

The problem seems to be the info.plist isnt right, as even (i just uploaded it for test flight) NOT App Store Submission)... The error upload says is the same....Info.plist does have the info for user access, but still something not right?
![Simulator Screen Shot - Apple Watch Series 6 - 44mm - 2020-09-26 at 23 05 19](https://user-images.githubusercontent.com/21694968/94351064-c8203300-004c-11eb-8a06-f1c19ff03a8c.png)
![Simulator Screen Shot - Apple Watch Series 6 - 44mm - 2020-09-26 at 23 05 11](https://user-images.githubusercontent.com/21694968/94351066-ca828d00-004c-11eb-9af0-ed1c47b6ded9.png)



<img width="721" alt="Screenshot 2020-09-26 at 23 04 19" src="https://user-images.githubusercontent.com/21694968/94351056-9effa280-004c-11eb-92d0-aa1fd329b07d.png">
<img width="729" alt="Screenshot 2020-09-26 at 23 04 09" src="https://user-images.githubusercontent.com/21694968/94351058-a4f58380-004c-11eb-9a7c-921c2ce50c8d.png">

**Comment by andyferriby** [comment]
Also please note, I AM NOT submitting to the app store, i did that to show you the error if you understand :-)

**Comment by BastiaanJansen** [comment]
Thank you for the screenshots! I created a new release with the fix! Can you let me know if it works?

**Comment by andyferriby** [comment]
Nope just the same with uploading for testflight, and ive not changed anything in the info.plists... Very odd, Maybe its xcode version 12.0.1<img width="741" alt="Screenshot 2020-09-27 at 10 20 39" src="https://user-images.githubusercontent.com/21694968/94361302-19f7a600-00ab-11eb-9a54-0bf878e6355d.png">

**Comment by BastiaanJansen** [comment]
It could be the XCode version. I just tested it on the beta version of XCode 12.2. The app runs fine on my Apple Watch with WatchOS 7 and iPhone on iOS 14.2.

**Comment by andyferriby** [comment]
See what happens if you try to upload to itunes connect?

**Comment by BastiaanJansen** [comment]
I don't have a paid developer account.
